trezor-crypto/src/bip32.c

```diff
@@ -64,6 +65,15 @@ const curve_info ed25519_cardano_info = {
 	.hasher_script = HASHER_SHA2,
 };
 
+const curve_info ed25519_blake2b_nano_info = {
+	.bip32_name = "ed25519 seed",
```

Comment:
Just a quick remark. This `bip32_name` value is the reason why the curve is named `"ed25519-blake2b-nano"` not `"ed25519-blake2b"`. _trezor-crypto_ convention seems to be to use _ed25519-xxx seed_ like values here (like below). The reason why Nano violates that convention here is provide interoperability between Ledger and trezor-crypto. At the time of the Ledger Nano app release Ledger SDK only supported _ed25519 seed_ value and I was blissfully unaware of the trezor-crypto convention. So now Nano needs to be the odd one out in trezor-wallet codebases.